scripts/testing/setup_private_registry.bat

```diff
@@ -42,8 +42,8 @@ call docker run ^
   -e "REGISTRY_AUTH_HTPASSWD_PATH=/auth/htpasswd" ^
   -v %~dp0certs:/certs ^
   -e "REGISTRY_HTTP_TLS_CERTIFICATE=/certs/registry_auth.crt" ^
-  -e "REGISTRY_HTTP_TLS_KEY=/certs/registry_auth.key" ^
-  registry:2
+  -e "REGISTRY_HTTP_TLS_KEY=/certs/registry_authssss.key" ^
```

Comment:
What's this change?

Reply:
ops, sorry. my fault. I'll change back

Reply:
one note. I changed the image from registry:2 to registry:2.7.0. I was getting an error from Windows with registry:2.

Reply:
I don't use Windows, so unfortunately it is difficult for me to review this.
Would you tell us the error log?

Reply:
This was the issuse: docker local registry “exec: \”htpasswd\“: executable file not found in $PATH”

And I was find a solution from this articles in stackoverflow:

https://stackoverflow.com/questions/62531462/docker-local-registry-exec-htpasswd-executable-file-not-found-in-path

Reply:
Yes, we pinned the registry to `2.7.0` due to this error for unix as well. We'd probably think of doing test at `appveyor` @suzuki-shunsuke